codewiki/src/be/dependency_analyzer/utils/logging_config.py

```
@@ -34,110 +34,110 @@
     COLORS = {
-        'DEBUG': Fore.BLUE,
-        'INFO': Fore.CYAN,
-        'WARNING': Fore.YELLOW,
-        'ERROR': Fore.RED,
-        'CRITICAL': Fore.RED + Style.BRIGHT,
+        "DEBUG": Fore.BLUE,
+        "INFO": Fore.CYAN,
+        "WARNING": Fore.YELLOW,
+        "ERROR": Fore.RED,
+        "CRITICAL": Fore.RED + Style.BRIGHT,
     }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)